py/desitarget/mock/mockmaker.py

```diff
@@ -445,11 +565,11 @@ def scatter_photometry(self, data, truth, targets, indx=None,
 
         # WISE sources are all point sources
         for band in ('W1', 'W2'):
-            fluxkey = 'FLUX_{}'.format(band)
-            ivarkey = 'FLUX_IVAR_{}'.format(band)
+            fluxkey  = 'FLUX_{}'.format(band)
+            ivarkey  = 'FLUX_IVAR_{}'.format(band)
             depthkey = 'PSFDEPTH_{}'.format(band)
 
-            sigma = 1 / np.sqrt(data[depthkey][indx]) / 5 # nanomaggies, 1-sigma
+            sigma    = 1 / np.sqrt(data[depthkey][indx]) / 5 # nanomaggies, 1-sigma
             targets[fluxkey][:] = truth[fluxkey] + rand.normal(scale=sigma)
```

Comment:
Cf my previous comment. There was no need to change existing, functional code to a different style.